dietpi/dietpi-software

```diff
@@ -2989,6 +3005,15 @@ _EOF_
 
 		fi
 
+		software_id=194 # PostgreSQL
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			G_AGI postgresql
```

Comment:
Do we want to put the data directory into DietPi userdata, like we do for MariaDB? I guess a symlink will work just the same way.